cypress/integration/pages/companies/edit_spec.page.js

```diff
@@ -0,0 +1,87 @@
+describe('Companies_Edition', () => {
+  const manager ='manager@example.com', password = '123456'
+  
+  const setup = () => {
+    cy.login_manager(manager, password)
+    cy.findAllByText('Empresa').click();
+    cy.findByText('Editar empresa').should('be.visible');
```

Comment:
I think it might be moved away from the setup, the setup step must be used to prepare for the expectation one. Maybe this can help: https://thoughtbot.com/blog/four-phase-test